code/game/objects/items/food/snacks.dm

```diff
@@ -163,3 +163,13 @@
 	tastes = list("pure electricity" = 3, "fitness" = 2)
 	foodtypes = TOXIC
 	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/bluespacebar
+    name = "Bluespace Bar"
```

Comment:
This indentation is not consistent with the rest of the file, use tabs.

Reply:
I'll look into it!

Reply:
![image](https://user-images.githubusercontent.com/76535113/103084678-849b7d80-45d7-11eb-8c40-2bddba381c6e.png)
I'm not 100% sure what the issue you're referring to is

Reply:
Welp, I think I fixed the problem